FrontEnd/src/components/basics/WizardForm/WizardForm.tsx

```diff
@@ -100,15 +104,28 @@ const WizardForm = <T extends {}>({
     );
   };
 
+  /**
+   * validatePickedValues is used to validate toParse using the provided schema. It
+   * only validates the keys provided in toValidate. This means that you don't have to
+   * validate everything inside of toParse (this is helpful so that you don't validate
+   * an entire page of the wizard form when only one input is changed).
+   *
+   * @param schema is the schema used to validate toParse
+   * @param toParse holds the values that should be validated/parsed
+   * @param toValidate is an array of keys within toParse that should be validated/parsed
+   * @returns
+   */
   const validatePickedValues = <P extends {} | unknown = unknown>(
```

Comment:
Would be helpful to write a comment to explain why. I couldn't seem to understand what it means for more than half an hour lol. Maybe explain what it means and why it is necessary. Shouldn't extends unknown be enough?

Reply:
I wish I added a comment to this originally, I don't remember why I did this lmao

Reply:
tested and ended up changing it to:
<P extends Partial<T>>